.babelrc

```diff
@@ -1,19 +1,19 @@
 {
   "presets": [
-    "env",
-    "react"
+    "@babel/preset-env",
+    "@babel/preset-react"
   ],
   "plugins": [
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-runtime"
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-transform-runtime"
   ],
   "env": {
     "test": {
       "presets": [
-        "es2015",
+        "@babel/preset-es2015",
```

Comment:
Obviously an existing issue, but since you're using `preset-env` already, I'd be very surprised if this was necessary.

Reply:
Good catch, we can solve it differently 👍

Reply:
Fixed 👌